### ISSUE TITLE
Getting weird overlays now

### ISSUE THREAD

**Issue by dessalines** [donkeystep/jira-dark#12]
Not sure why, but I'm getting some really strange overlays from boards pages.

**Comment by sergey124** [comment]
Hello @dessalines , maybe you could give screenshots of the part of the board having issues?

I tried to reproduce in my project board, but it's not hosted at jira.atlassian.com and has additional custom styles, so it might look different.
I couldn't find open-to-the-world boards hosted at jira.atlassian.com, if you know any, I'd also be grateful.

**Comment by dessalines** [comment]
![screenshot from 2017-04-06 12-06-01_modified](https://cloud.githubusercontent.com/assets/930215/24771478/959e43fe-1ac2-11e7-8665-f20ede7b5cff.png)

**Comment by sergey124** [comment]
@dessalines , maybe I get it wrong, do I understand you correctly that the issue is this dotted line border around "Backlog" and "Plan a sprint..."?
Or are these P- and T-shaped black areas actually the issue? ( Means it's not some sensitive data removed.)

**Comment by dessalines** [comment]
They kinda happen all over the page, not just on that top bar.

**Comment by sergey124** [comment]
@dessalines , so is the issue about dotted line border or black areas?

**Comment by dessalines** [comment]
The black areas. My guess is that jira is doing some funky things with overlays that this theme is styling as black.

**Comment by sergey124** [comment]
Yes, black areas cover functional elements, that's bad. 
Would be great to reproduce somewhere public.

Is this JIRA hosted in jira.atlassian.com or separate instance?

**Comment by dessalines** [comment]
I agree, a place to test would be nice, but its on a private jira. I tried to find an open one, but couldn't replicate the error. 

Another weird thing is when I open up the chrome inspector, the problem goes away, and the overlays correctly show. 

Since we don't have a test area, you can close out this one, it's an edge case that we can't test to see what's going on.
